src/faebryk/exporters/pcb/kicad/transformer.py

```diff
@@ -259,9 +261,12 @@ def attach(self):
 
             pin_names = g_fp.get_trait(has_kicad_footprint).get_pin_names()
             for fpad in g_fp.IFs.get_all():
-                pad = find(
-                    fp.pads, lambda p: p.name == pin_names[cast_assert(FPad, fpad)]
-                )
+                try:
+                    pad = find(
+                        fp.pads, lambda p: p.name == pin_names[cast_assert(FPad, fpad)]
+                    )
+                except KeyErrorAmbiguous:
+                    continue
```

Comment:
This will ignore all f_pads that have multiple pcb_pads.
This is not ok I think.

Reply:
```python
              try:
                  pad = find(
                      fp.pads, lambda p: p.name == pin_names[cast_assert(FPad, fpad)]
                  )
              except KeyErrorAmbiguous as e:
                  pad = e.duplicates[0]
```

This is slightly better, but still not good.
Better would be to update the trait to accept 1..n pads